oh-my-bash.sh

```diff
@@ -8,7 +8,7 @@ esac
 
 # Check for updates on initial load...
 if [ "$DISABLE_AUTO_UPDATE" != "true" ]; then
-  env OSH=$OSH DISABLE_UPDATE_PROMPT=$DISABLE_UPDATE_PROMPT bash -f $OSH/tools/check_for_upgrade.sh
+  OSH=$OSH DISABLE_UPDATE_PROMPT=$DISABLE_UPDATE_PROMPT source $OSH/tools/check_for_upgrade.sh
```

Comment:
Sourcing `check_for_upgrade.sh` results in importing functions and variables inside the script. I think we need to rename these functions and variables to avoid possible conflicts.